inginious/frontend/templates/course_admin/api_tokens.html

```diff
@@ -0,0 +1,94 @@
+<h2>{{_("API Tokens")}}</h2>
+<div class="card-header bg-success text-white">
+    {{ _("Add token") }}
+</div>
+<div class="card-body">
+    <form class="form-horizontal" id="tokens_form" method="post">
+        <div class="row">
+            <div class="col-md-6">
+                <input type="text" placeholder="description" name="descr" value="" class="form-control">
+            </div>
+            <div class="col-md-3">
+                <div class="form-group">
+                    <div class='input-group date' id='time_picker' data-target-input="nearest">
+                        <input data-target='#time_picker' name="expiration" data-date-format="YYYY-MM-DD HH:mm:ss" placeholder="2014-06-29 10:00" type='text' class="form-control datetimepicker-input"/>
+                        <div class="input-group-append" data-target="#time_picker" data-toggle="datetimepicker">
+                            <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-3">
+                <button class="btn btn-block btn-info" type="submit" name="add_token"><i class="fa fa-plus fa-lg"></i> {{ _("Create token") }}</button>
+            </div>
+        </div>
+    </form>
+    <script type="text/javascript">
+        $(function() {
+            $('#time_picker').datetimepicker({locale: '{{user_manager.session_language()}}', sideBySide: true, format:'YYYY-MM-DD HH:mm:ss'});
+        });
+    </script>
+     {% if msg != "" and msg != "removed" %}
+        <h4>{{ _("Your token string is shown under here. You can now copy it where you need it. It will not be possible to recover the string afterward.") }}</h4>
+        <hr>
+        <p>{{msg}}</p>
+    {% endif %}
+</div>
+<div class="card-header bg-dark text-white">
+    {{ _("Existing tokens") }}
+</div>
+<div class="card-body">
+    <form class="form-horizontal" id="remove_form" method="post">
+        <table id="table" class="table">
+            <tbody>
+                {% if msg == "removed" %}
+                    <p>{{ _("Token removed") }}</p>
+                {% endif %}
+                {% for token in tokens %}
+                    {% set tok_id = token["_id"] %}
+                    <tr id="{{loop.index}}">
+                        <td>ID : {{token["_id"]}}</td>
+                        {% if token["description"] == "" %}
+                            <td class="text-secondary">{{ _("No description") }}</td>
+                        {% else %}
+                            <td>{{token["description"]}}</td>
+                        {% endif %}
+                        {% if str(token["expire"]) == "9999-12-31 00:00:00" %}
+                            <td class="text-secondary">{{ _("Doesn't expire") }}</td>
+                        {% elif token["expire"] >= now %}
+                            <td>Expire {{token["expire"].strftime("%Y %m %d %H:%M:%S")}}</td>
```

Comment:
Why don't you use the same converter ? `str` or, as documented, `isoformat(' ')` ?